themes/javascript.tpl

```diff
@@ -51,7 +51,7 @@ var {$k} = '{$def|@addcslashes:'\''}';
 {/if}
 {if isset($js_files) && $js_files|@count}
 {foreach from=$js_files key=k item=js_uri}
-<script type="text/javascript" src="{$js_uri}"></script>
+<script type="text/javascript" src="{$js_uri}?version={$smarty.const._TB_VERSION_}"></script>
```

Comment:
This will not work in all cases, as {$js_uri} can already contain `?` character. That would lead to incorrect URL with two `?` in it.

Anyway, I think it's not a best idea to do this in template. Better way would be to override `addJS` method in `AdminController`, and provide version string there (based on asset modification date, for example)